README.md

```diff
@@ -1,3 +1,65 @@
+## Project title
+* Takeaway Challenge
+
+## Motivation
+
+Client request for a program that can perform the following functions.
+* Print a list of dishes with prices.
+* Select dishes from the list using numbers.
+* Check the number of dishes and the cost.
+* Receive a thank you message with the delivery time.
+* Bonus: Send a text message with the thank you message and the deliveery time.
+
+## Build status
+* Initial files being created and dependencies installed
+* Wrote the menu_spec.rb file that contained a tests.
+* Created the menu.rb file with the menu class that holds the dishes. 
+* Wrote the takeaway_spec.rb file that contained the tests. This also has a double for the dishes.
+* Created the takeaway.rb file that has the functionality to print the menu, select dishes with a number and confirm the order with a text.
+* This code passes all the allocated tests and the text message system worked. Howeverv, I do not know how to write a test for the send_text method.
+* My account details for the send_text method have been removed.
```

Comment:
![image](https://user-images.githubusercontent.com/75490778/117640802-516c1c00-b17d-11eb-9b5f-88158ccee13a.png) 
Congrats on getting the SMS confirmation implemented! Massive achievement to get a third party API interacting with your code.